web/src/components/auth/SignUpSnackbar.test.tsx

```diff
@@ -30,64 +40,143 @@ const setLargeScreen = (value: boolean): void => {
   });
 };
 
-describe("SignUpSnackbar", () => {
+describe("<SignUpSnackbar />", () => {
   beforeEach(() => {
     jest.resetAllMocks();
     useMockRouter({});
-    useMockBusiness({});
+    setupStatefulUserDataContext();
   });
 
   const setRegistrationAlertIsVisible = jest.fn();
-  const setupHookWithAuth = (isAuthenticated: IsAuthenticated, registrationAlertIsVisible = true): void => {
+
+  const renderWithAuth = ({
+    isAuthenticated,
+    registrationAlertIsVisible,
+  }: {
+    isAuthenticated: IsAuthenticated;
+    registrationAlertIsVisible?: boolean | undefined;
+  }): void => {
+    render(
+      withAuthAlert(
+        <WithStatefulUserData initialUserData={generateUserData({})}>
+          <SignUpSnackbar />
+        </WithStatefulUserData>,
+        isAuthenticated,
+        {
+          registrationAlertIsVisible: registrationAlertIsVisible ?? true,
+          setRegistrationAlertIsVisible,
+        }
+      )
+    );
+  };
+
+  const renderWithAuthAndUser = ({
+    isAuthenticated,
+    activeUser,
+  }: {
+    isAuthenticated: IsAuthenticated;
+    activeUser: ActiveUser;
+  }): void => {
     render(
-      withAuthAlert(<SignUpSnackbar />, isAuthenticated, {
-        registrationAlertIsVisible,
-        setRegistrationAlertIsVisible,
-      })
+      withAuth(
+        withAuthAlert(
+          <WithStatefulUserData initialUserData={generateUserData({})}>
+            <SignUpSnackbar />
+          </WithStatefulUserData>,
+          isAuthenticated,
+          {
+            registrationAlertIsVisible: true,
+            setRegistrationAlertIsVisible,
+          }
+        ),
+        { activeUser, isAuthenticated }
+      )
     );
   };
```

Comment:
These 2 functions seem pretty similar. I don't really mind them being distinct given this is a test file and I personally like being more verbose when it comes to tests, but I feel like there is enough overlap here that we could combine if we wanted.